docs/content/Getting-Started/Cloud/03-Generate-models.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Generate models
+permalink: /getting-started/cloud/generate-models
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
```

Comment:
Again, a really nice place to have a paragraph explaining why should one do that.